docs/Reference/Responsible-Disclosure.md

```diff
@@ -8,4 +8,4 @@ there may still be vulnerabilities present.
 If you discover a vulnerability, we want to know about it so we can take steps to address it as 
 quickly as possible. We would like to ask you to help us better protect our clients and our systems.
  
-Please follow the process explained at https://wiki.hyperledger.org/display/HYP/Defect+Response
\ No newline at end of file
+Please follow the process explained on [Hyperledger defect response wiki page](https://wiki.hyperledger.org/display/SEC/Defect+Response)
```

Comment:
Minor suggestion, add "the". 

For example, "Please follow the process explained on the ..."